pkg/commands/drift/command.go

```diff
@@ -17,8 +17,11 @@ package drift
 import (
 	"context"
 	"fmt"
+	"sort"
 	"strings"
 
+	"golang.org/x/exp/maps"
```

Comment:
Should we use https://pkg.go.dev/maps now rather than https://pkg.go.dev/golang.org/x/exp/maps? I think so, but maybe there's a reason I'm missing.

Reply:
Done

Reply:
Actually, reverted because maps and slices packages are not available until 1.23. We are on 1.22.